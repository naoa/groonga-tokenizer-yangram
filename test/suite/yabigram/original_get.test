register tokenizers/yangram

tokenize TokenYaBigram "アイＡＢうえ" NormalizerAuto --mode GET
