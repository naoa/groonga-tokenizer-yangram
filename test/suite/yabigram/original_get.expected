register tokenizers/yangram
[[0,0.0,0.0],true]
tokenize TokenYaBigram "アイＡＢうえ" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "アイ",
      "position": 0
    },
    {
      "value": "ab",
      "position": 2
    },
    {
      "value": "うえ",
      "position": 3
    }
  ]
]
